docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,60 @@
+## Math Formulae
+Markbind supports typesetting math equations with [KaTeX](https://katex.org).
```

Comment:
Since `katex` is the rendering engine, not the spec, let's start with the spec first, then list the engine's use